docs/process/metrics.md

```diff
@@ -53,3 +53,5 @@ These are general metrics about feature usage and specific feature behaviors. Th
 | `mergedWithLoadingHintCount` | The number of times the user merged before seeing the result of the merge hint. | To understand how many people are merging before learning whether there will be conflicts or not |
 | `mergedWithCleanMergeHintCount` | The number of times the user has merged after seeing the 'no conflicts' merge hint. | To understand how many "clean" merges there are |
 | `mergedWithConflictWarningHintCount` | The number of times the user has merged after seeing the 'you have XX conflicted files' warning. | To understand how frequently people are merging even though they know there will be conflicts |
+| `mergedSuccesfulAfterConflictsCount` | The number of times the user successfully completes a merge after a merge conflict. | To understand how effectively users are able to resolve conflicts and complete their merge successfully |
```

Comment:
I know we haven't specified these to be gramatically correct but `mergedSuccessful` is messing with me. Could this be `mergedSuccessfullyAfterConflictsCount`? Or maybe just `mergeSuccessAfterConflictsCount`?

Reply:
`successfulConflictResolutionCount`?